launch/world_modeling/world_model.launch

```diff
@@ -1,54 +1,46 @@
 <?xml version="1.0"?>
 <launch>
 
-    <!-- Get the machine file -->
-    <arg name="machine" default="localhost"/>
-    <arg name="robot_env" default="$(optenv ROBOT_ENV robotics_testlabs)" />
-    <arg name="robot_real" value="$(optenv ROBOT_REAL false)" />
+	<!-- Get the machine file -->
+	<arg name="machine" default="localhost"/>
+	<arg name="robot_env" default="$(optenv ROBOT_ENV robotics_testlabs)" />
+	<arg name="robot_real" value="$(optenv ROBOT_REAL false)" />
 
-    <include file="$(env ROBOT_BRINGUP_PATH)/machines/$(arg machine).machine" />
+	<include file="$(env ROBOT_BRINGUP_PATH)/machines/$(arg machine).machine" />
 
-    <arg name="config" default="world_modeling/world_model.yaml"/>
+	<arg name="config" default="world_modeling/world_model.yaml"/>
 
-    <!-- World model -->
-    <node name="ed" pkg="ed" type="ed" output="log" machine="$(arg machine)" />
+	<!-- World model -->
+	<node name="ed" pkg="ed" type="ed" output="log" machine="$(arg machine)" />
 
-    <!-- Configure world model -->
-    <node name="ed_configure" pkg="ed" type="configure" args="$(env ROBOT_BRINGUP_PATH)/parameters/$(arg config)" output="log" machine="$(arg machine)"/>
+	<!-- Configure world model -->
+	<node name="ed_configure" pkg="ed" type="configure" args="$(env ROBOT_BRINGUP_PATH)/parameters/$(arg config)" output="log" machine="$(arg machine)"/>
 
-    <!-- Face recognition -->
-    <group if="$(arg robot_real)">
+	<!-- Face recognition -->
+	<group ns="face_recognition">
```

Comment:
Is this correct? I understand thr ns-addition, but not removal of the check for robot_real.